app/views/dashboard_v2/dashboard.html.erb

```diff
@@ -0,0 +1,87 @@
+<%# TODO: Topics to display if the user is new and not yet subscribed to other posts %>
```

Comment:
The design showed that a new user will get an update saying "We have started you off with a few topics"
These topics should be featured. I currently don't know if there is a query I can use to show this

Reply:
Hi Ruth, as of the final discussion in https://github.com/publiclab/plots2/issues/6072#issuecomment-624275876, we are not pre-selecting any topics, so we can skip that portion of the welcome message. We should just include the blog, and rely on the bottom-of-the-column featured and trending topics to encourage new people to begin subscribing. Let's see how that looks and if it's exceptionally weird we can discuss further changes! But, for now the query isn't required. 

Or did you mean a query to see if the user is new? I believe there is a conditional on the current dashboard which we can use for this.

Reply:
Thanks for the clarification, it was mostly about the topics to display if it's a new user. 
About the `bottom-of-the-column featured and trending topics` should I remove any topics that a user has subscribed to from that section? Let's say that the user has subscribed to a topic "test" and test is either listed in the trending or featured topics, should it be removed, or can it stay the same?

Reply:
For trending topics i think we can remove already subscribed ones, but for featured, i think it'd be too tough to work with that if it's manually written... We'd have to use some brittle Javascript or something, and it's probably not worth it. 

Great!